hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/DatanodeTable.java

```diff
@@ -65,6 +65,11 @@ public void delete(KEY key) throws IOException {
     table.delete(key);
   }
 
+  @Override
```

Comment:
We don't need changes to provide this API on data nodes.

Reply:
It's true that DN won't use it at the moment but I had to put it here anyway because it implements `Table`. Plus it also has `getRangeKVs()`.